conda/activate.py

```diff
@@ -23,21 +24,31 @@ class Activator(object):
     #   2. Execute/source activate.d/deactivate.d scripts
     #   3. Update the command prompt
     #
-    # Shells should also use 'reactivate' following conda's install, update, and
-    #   remove/uninstall commands.
+    # Shells should also use 'reactivate' following conda's install, update,
+    # and remove/uninstall commands.
     #
-    # All core logic is in build_activate() or build_deactivate(), and is independent of
-    # shell type.  Each returns a map containing the keys:
-    #   export_vars
-    #   unset_var
-    #   activate_scripts
-    #   deactivate_scripts
+    # All core logic is in the build_*() functions:
+    #   - build_activate()
+    #   - build_deactivate()
+    #   - build_deactivatepost()
+    #   - build_reactivate()
+    #   - build_reactivatepost()
+    # and is independent of shell type.  Each returns a map optionally
+    # containing the keys:
+    #   set_vars            (dict of key-values to set)
+    #   export_vars         (dict of key-values to export)
+    #   unset_vars          (tuple of keys to unset)
+    #   unexport_vars       (tuple of keys to un-export)
+    #   activate_scripts    (tuple of activate scripts)
+    #   deactivate_scripts  (tuple of deactivate scripts)
+    #   deactivatepost      (bool to include deactivate post command)
+    #   reactivatepost      (bool to include reactivate post command)
     #
-    # The value of the CONDA_PROMPT_MODIFIER environment variable holds conda's contribution
```

Comment:
The prevalent (and linter-checked, https://github.com/conda/conda/blob/4.5.0rc0/setup.cfg#L34) style in the code base uses a 99 character line limit. Please don't enforce foreign rules and revert all those changes.

Reply:
Didn't realize conda wasn't using the 78 char convention.